src/helpers/build-image-service-url.js

```diff
@@ -1,12 +1,15 @@
 import qs from 'querystring';
 
+const capiRX = /^http:\/\/com\.ft\.imagepublish\.prod\.s3\.amazonaws\.com\/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/
```

Comment:
The regex will need to be `.prod(-us)?` in place of `.prod` as there are two s3 buckets, one in eu and one in us.